layouts/index.html

```diff
@@ -24,74 +24,39 @@ <h1>{{ .Title}}</h1>
 			</div>
 		</header>
 
-
 		<h1 id="products">{{ $.Param "products" }}  🚀</h1>
 
-		<section class="row">
-			<div class="left-col">
-				<h3>{{ $.Param "usecases.mes_aides.title" }}</h3>
-				<p>{{ $.Param "usecases.mes_aides.description" }}</p>
-			</div>
-
-			<figure>
-				<img class="screenshot" src="/img/mes-aides.png" alt="Mes Aides">
-				<figcaption><a href="https://beta.gouv.fr/startup/mes-aides">Mes Aides</a> {{ $.Param "usecases.mes_aides.caption_1" }} <a href="https://openfisca.org/doc/openfisca-web-api/index.html">Web API</a> {{ $.Param "usecases.mes_aides.caption_2" }}</figcaption>
-			</figure>
-		</section>
-
-		<section class="row">
-			<div class="left-col"></div>
-
-			<figure>
-				<img class="screenshot" src="/img/les-meves-ajudes.png" alt="Les mes ajudes">
-				<figcaption><a href="https://ajuntament.barcelona.cat/lesmevesajudes/">Les meves ajudes</a> {{ $.Param "usecases.les_meves_ajudes.caption_1" }} <a href="https://openfisca.org/doc/openfisca-web-api/index.html">Web API</a> {{ $.Param "usecases.les_meves_ajudes.caption_2" }}</figcaption>
-			</figure>
-		</section>
-
-		<section class="row">
-			<div class="left-col">
-				<h3>{{ $.Param "usecases.leximpact.title" }}</h3>
-				<p>{{ $.Param "usecases.leximpact.description" }}</p>
-			</div>
-
-			<figure>
-				<img class="screenshot" src="/img/leximpact.png" alt="LexImpact">
-				<figcaption><a href="https://leximpact.beta.gouv.fr/">LexImpact</a> {{ $.Param "usecases.leximpact.caption_1" }} <a href="https://openfisca.org/doc/openfisca-python-api/index.html">Python API</a> {{ $.Param "usecases.leximpact.caption_2" }}</figcaption>
-			</figure>
-		</section>
-
-		<section class="row">
-			<div class="left-col">
-				<h3>{{ $.Param "usecases.reforms.title" }}</h3>
-				<p>{{ $.Param "usecases.reforms.description" }}</p>
-			</div>
-
-			<figure>
-				<img class="screenshot" src="/img/revenu-de-base.jpg" alt="Comment appréhender le coût d’un revenu de base ?">
```

Comment:
`/img/revenu-de-base.jpg` seems to have been duplicated into `/static/img/showcase/revenu-de-base.jpg` instead of being moved 😉

Reply:
Sorry about that.
I'll have to rewrite the commit before merge, right?

Reply:
Yes, that would save storage.